### PR TITLE
Org-related changes

### PR DESCRIPTION
**Pull request by CoralineAda**

### CODE REVIEW

File: content/apply_org.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
++++
+title = "Join the Movement"
+heading = "Join the Organization for Ethical Source"
++++
+
+If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indication the membership level that you are interested in:
+
+* _Active Members_ are individuals who volunteer their time, expertise, and talent towards the work of the organization. Active members have voting rights. Membership fees for active members are waived.
+* _Supporting Members_ are individuals or organizations who support the work of the organization through their membership fees. Supporting members do not have voting rights. Annual membership fees are $50 in the US, £50 in the UK, €50 in the European Union, or the equivalent in their local currencies.
```
Comment (tobie) [inline content/apply_org.md]:
```suggestion
* _Supporting Members_ are individuals or organizations who support the work of the organization through their membership fees. Supporting members do not have voting rights. Annual membership fees are 50€ in the European Union, ¥5,000 in Japan, CHF50 in Switzerland, and $50 in the US and elsewhere.
```

File: content/apply_org.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+* _Associate Members_ are noncommercial organizations whose activities are related to our mission. Associate members do not pay membership fees and do not have voting rights.
```
Comment (tobie) [inline content/apply_org.md]:
```suggestion
```

I would not collect info or speak about Associate members until we have a clear program for them.

File: content/apply_org.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
++++
+title = "Join the Movement"
+heading = "Join the Organization for Ethical Source"
++++
+
+If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indication the membership level that you are interested in:
```
Comment (tobie) [inline content/apply_org.md]:
```suggestion
If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indicate the kind of membership that you are interested in:
```

Let's not talk about levels, here. These are different ways of helping. All are valid.

File: content/apply.md [hunk 4]
```diff
@@ -1,42 +1,9 @@
 +++
-title = "Join the Ethical Source Working Group"
-heading = "Sign up to join the Ethical Source working group."
+title = "Join the Movement"
 +++
 
-The Ethical Source working group is a community for open source maintainers and contributors, tech ethicists and advocates, licensing experts, and more to come together to strategize about ways to incorporate ethics into our open source work.
+### Join the Organization for Ethical Source
+If you are interested in joining the Organization for Ethical Source as an active, supporting, or associate member, please [use this form](/apply_org) to get in touch with us. All members of the organization are automatically invited to participate in the Working Group.
```
Comment (tobie) [inline content/apply.md]:
```suggestion
If you are interested in joining the Organization for Ethical Source as an active or support member, please [use this form](/apply_org) to get in touch with us.
```

* Removes the mention to associate member (see rationale below)
* Removes the last sentence that mentions the working group, as it ads confusion; active members will be invited to join the WG, support members won't.

File: content/apply_org.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+        <p>E-mail</p>
+        <input name="email" type="email" required>
+      </label>
+    </div>
+    <div class="form-group">
+    <label>
+      <p>Membership Type</p>
+      <input name="membership_level" type="radio" value="Active Member">Active Member<br />
+      <input name="membership_level" type="radio" value="Supporting Member">Supporting Member<br />
+      <input name="membership_level" type="radio" value="Associate Member">Associate Member<br />
```
Comment (tobie) [inline content/apply_org.md]:
```suggestion
```

File: content/apply_org.md [hunk 6]
```diff
@@ -0,0 +1,50 @@
...
+    </label>
+    </div>
+    <div class="form-group">
+      <label>
+        <p>Message, brief bio, and why you're interested</p>
+        <textarea name="message" required></textarea>
+      </label>
+    <fieldset>
+      <button type="submit" class="button">Send</button>
```
Comment (tobie) [inline content/apply_org.md]: We should add a checkbox here for compliance with GDPR.

- [ ] I agree to received updates and information about the ESO and our activities. We won't share your email address without your express permission ever.

File: content/apply_org.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
++++
+title = "Join the Movement"
+heading = "Join the Organization for Ethical Source"
++++
+
+If you are interested in joining the Organization for Ethical Source as a member, please fill out the form below. Please indication the membership level that you are interested in:
+
+* _Active Members_ are individuals who volunteer their time, expertise, and talent towards the work of the organization. Active members have voting rights. Membership fees for active members are waived.
```
Comment (tobie) [inline content/apply_org.md]:
```suggestion
* _Active Members_ are individuals who volunteer their time, expertise, and talent towards the work of the organization. Active members have voting rights and don not pay membership fees. Active members will be invited to join our Slack instance where most of the discussions are taking place.
```

**Review by tobie — COMMENTED** [review]
A few suggestions inline.

File: content/apply_org.md (hunk 3 above)
Comment (CoralineAda) [reply content/apply_org.md]: Not sure what you mean here besides "kind" over "level".

Comment (tobie) [reply content/apply_org.md]: That was just additional context.

**Review by BICKDICK — APPROVED** [review]

**Comment by tobie** [comment]
The other option would be to link the bylaws from the footer of the page.

**Comment by CoralineAda** [comment]
@tobie I think I figured it out! I added a 'Governance' link to the nav. The Governance page has a heading for the org and a heading for the working group. Under the first is a link to bylaws in English and French, and under the other a link to the code of conduct (also English and French versions). I think it looks pretty good.

**Comment by tobie** [comment]
That sounds good to me from the perspective of keeping the navigation clean, but aren’t you concerned it makes the CoC less findable by someone who would really need to find it?

**Comment by CoralineAda** [comment]
The code of conduct was only ever linked in the working group application page, so I think this makes it more findable than before.

**Pull request merged**